clean.cmd

```diff
@@ -1,105 +1,42 @@
 @if not defined _echo @echo off
 setlocal EnableDelayedExpansion
 
-echo Running clean.cmd
+set NO_DASHES_ARG=%1
+if /I [%NO_DASHES_ARG:-=%] == [?] goto Usage
```

Comment:
Thanks for adding this. Can you do the same in corefx?

Reply:
Yes

Reply:
Just checked that it is there - so it seems that rather I copied it here

Reply:
Hmmm. I don't see it in corefx https://github.com/dotnet/corefx/blob/master/clean.cmd.

Reply:
I deleted my comment - misread lines of code 😄